sms_wsj/database/wsj/create_json.py

```diff
@@ -8,6 +8,7 @@
 import os
 import re
 import tempfile
+import subprocess
```

Comment:
`import sh` is now unused

Reply:
Can you also fix the imports for the scipy window functions (in the stft)? Currently, some Import tests still fail.